packages/pyolite-kernel/src/declarations.d.ts

```diff
@@ -14,3 +14,12 @@ declare let _pipliteUrls: string[];
 declare let _disablePyPIFallback: boolean;
 declare let pyodide: any;
 declare let loadPyodide: any;
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+interface WindowOrWorkerGlobalScope {
```

Comment:
We synced the typescript version with the upstream, so you should be able to remove the extra typing.